### PR TITLE
(BOLT-1133) Add slack channel info to documentation

### PR DESCRIPTION
**Pull request by lucywyman**
This adds our Slack channel info to the README, the main `bolt` documentation page, and the `installing Bolt` page. Also adds Puppet's twitter to the `bolt` docs page, and updates verbage of the CONTRIBUTING doc for consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ Additionally the Bolt project includes:
     * [Writing Plans](https://puppet.com/docs/bolt/latest/writing_plans.html)
 * [Applying Manifest Blocks in Plans](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html)
 
+## Getting Help
+
+* [#bolt on Slack](https://slack.puppet.com/) - Join the Bolt developers and community
```
Comment (donoghuc) [inline README.md]: the `#` is rendered in the link

File: pre-docs/bolt.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ Bolt is an open source task runner that automates the manual work that you do to
 
  -   **Share and contribute**
 
+[Join us on Slack](https://slack.puppet.com/) - Join the `#bolt` channel in the [Puppet community Slack](https://slack.puppet.com/) .
```
[the remark this answers is not in the document]
Comment (lucywyman) [reply pre-docs/bolt.md]: Oh woops yes!

File: README.md (hunk 1 above)
Comment (donoghuc) [reply README.md]: oh, intentionally. Slack channel are prefixed with `#`. my bad

**Comment by donoghuc** [comment]
@melamos I added `docs` label to ticket https://tickets.puppetlabs.com/browse/BOLT-1133 Is it Ok If I merge this in?

File: pre-docs/bolt.md [hunk 3]
```diff
@@ -71,6 +71,10 @@ Bolt is an open source task runner that automates the manual work that you do to
 
  -   **Share and contribute**
 
+[Join us on Slack](https://slack.puppet.com/) - Join the `#bolt` channel in the [Puppet community Slack](https://slack.puppet.com/)
```
Comment (melamos) [inline pre-docs/bolt.md]:
```suggestion
[Join us on Slack](https://slack.puppet.com/) - Join the `#bolt` channel
```

**Pull request merged**